jupyterlab_delux/labextensionapp.py

```diff
@@ -0,0 +1,12 @@
+import os
+import sys
+from jupyterlab import labextensions
+
+
+def main():
+    os.environ['JUPYTERLAB_DIR'] = os.path.join(sys.prefix, 'share', 'jupyter', 'delux')
+    labextensions.BaseExtensionApp.app_dir = os.environ['JUPYTERLAB_DIR']
+    labextensions.main()
+    
+if __name__ == '__main__':
+    sys.exit(main())
```

Comment:
Awesome 🎉

Reply:
And I did test it out and confirm that I can list the default extensions, install a new one, and list again and it all works like it should.